src/types.js

```diff
@@ -94,6 +94,11 @@ export type Descriptor = {
   set?: UndefinedValue | CallableObjectValue | AbstractValue,
 };
 
+export type FunctionBodyAstNode = {
+  // Function body ast node will have uniqueTag after interpreted.
+  uniqueTag: ?number,
```

Comment:
`uniqueTag?: number`. That represents the fact that this value can in fact be missing, and it can never be `null`.